.github/workflows/release-action.yaml

```diff
@@ -18,4 +19,4 @@ jobs:
       uses: ncipollo/release-action@v1
       with:
         bodyFile: "release/note.md"
-        tag: "v1.0.0"
+        tag: "v1.0.1"
```

Comment:
The code patch updates the GitHub actions configuration file to trigger the release creation on push to the master branch instead of the pull_request event. It also updates the tag version from v1.0.0 to v1.0.1. Overall, the code patch seems to be simple and straightforward with no significant risks or bugs. However, it may benefit from adding more checks and validations, such as making sure that the version number is always incremented and checking for any conflicts with existing tags. Additionally, documenting the release note markdown file format and how to update it would be helpful for future contributors.